app/assets/javascripts/modules/was_seed_viewer.js

```diff
@@ -1,9 +1,34 @@
+/*global Sly */
 
 (function( global ) {
   'use strict';
   var Module = (function() {
+
     return {
       init: function() {
+        var $wasThumbFrame = $('.sul-embed-was-seed');
+        var dataAttributes = $wasThumbFrame.data();
+
+        // Create the SLY scroll div on fly
+        var $wasThumbScroll = $('<div class="sul-embed-thumb-slider-scroll"><div class="sul-embed-thumb-slider-handle"/></div>');
```

Comment:
Also to note, it may be a way to speed up the tests ... for each individual feature test, it creates a lot of overhead. So for each `it` block the test framework is doing quite a bit just to test for the scrollbar for example. Testing the scrollbar as a feature test works, but may make sense later as a js test... #Tradeoffs :) happy to discuss in office tomorrow, but I definitely don't think it should block this PR.